sprintly/sprintly-kanban
# Search results layout

#### What's this PR do?

Adds more testing around the search interface
Decouples the search controls from the results interface to reduce flashes of content
Adds deterministic filter control states based on user query entry
#### Where should the reviewer start?

`app/views/pages/search.js`
#### How should this be manually tested?

Pull the app and play with the search controls
Enter this `type:story type:defect` into the query bar and observe the control states update to `active`
#### Any background context you want to provide?

N/A
#### What are the relevant tickets?

N/A
#### Screenshots (if appropriate)

![image](https://cloud.githubusercontent.com/assets/2892213/7328251/c525eed6-ea88-11e4-8353-6601224cba95.png)

![search_gif](http://g.recordit.co/37C7yM0a4V.gif)
#### How does this make you feel

![filter](http://i.giphy.com/FF7m8v3RrZgGI.gif)
